src/coreclr/inc/corhlprpriv.h

```diff
@@ -294,9 +294,9 @@ class CQuickMemoryBase
     }
 
     // Copy single byte string and hold it
-    const char * SetStringNoThrow(const char * pStr, SIZE_T len)
+    const char * SetString(const char * pStr, SIZE_T len)
     {
-        LPSTR buffer = (LPSTR) AllocNoThrow(len + 1);
+        LPSTR buffer = (LPSTR) AllocThrows(len + 1);
```

Comment:
sorry if this is wrong, but will `buffer` never be `NULL`, because if `AllocThrows` fails, it throws? So we can remove the `if` check below

Reply:
Nice catch. The if check below is superfluous.